# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/towitoko/c23-bool.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Use stdbool.h, bool is a keyword in C23.

--- a/src/driver/defines.h	2026-09-25 19:37:53.974697933 +0200
+++ b/src/driver/defines.h	2026-09-25 19:37:53.974697933 +0200
@@ -48,7 +48,7 @@
 
 typedef unsigned char BYTE;
 #ifndef __cplusplus
-typedef int bool;
+#include <stdbool.h>
 #endif
 typedef unsigned short USHORT;
 typedef unsigned char UCHAR;
